migrations/Version202607310755302234_tao.php

```
@@ -0,0 +1,38 @@
+use oat\tao\model\accessControl\func\AclProxy;
+use oat\tao\model\user\TaoRoles;
+use oat\tao\scripts\tools\migrations\AbstractMigration;
+
+final class Version202607310755302234_tao extends AbstractMigration
+{
+    public function getDescription(): string
+    {
+        return 'Apply roles to access tao/Log/logFrontendAction';
+    }
+
+    public function up(Schema $schema): void
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$schema'. (undefined)

(UnusedFormalParameter)